.github/workflows/ci-testing.yml

```diff
@@ -13,10 +13,16 @@ jobs:
         os: [ubuntu-latest, macos-latest, windows-latest]
         python-version: [3.9, '3.10', '3.11', '3.12']
         shapely-dev: [false]
+        no-network: [false]
         include:
           - os: ubuntu-latest
             python-version: '3.11'
             shapely-dev: true
+            no-network: false
```

Comment:
They do pick up the default.  The only difference was that without adding these explicitly both were marked in the test output as "ubuntu-latest, 3.11, false".  So having it like this made it easier for me to find the one I was looking for.  Future us cannot be expected to remember what the order of these booleans mean though, so will revert that.

Reply:
I'm also wondering if we still need the `shapely-dev` one, as it looks like it was added in the run up to the Shapely 2.0 release to make sure we were ready for that: #2080

Reply:
Yeah, that is a good point, it might make sense to keep it though and turn it into a monthly pre-release run to get all of Shapely+Numpy+Matplotlib nightlies instead and just run with everyone's updates once every so often rather than with everyone's PRs.

Maybe leave it as a follow-up PR cleaning some of this other stuff up so it doesn't blow up the scope of this one.

Reply:
Actually it turns out these flags default to `false`, so I do need `use-network: true` in `shapely-dev`.